pandas/tests/frame/methods/test_add_prefix_suffix.py

```diff
@@ -18,3 +18,22 @@ def test_add_prefix_suffix(float_frame):
     with_pct_suffix = float_frame.add_suffix("%")
     expected = Index([f"{c}%" for c in float_frame.columns])
     tm.assert_index_equal(with_pct_suffix.columns, expected)
+
+
+def test_add_prefix_suffix_axis(float_frame):
```

Comment:
Could you add a similar test for `Series` as well (may need to make a new file in `pandas/tests/series/methods/`?

Reply:
I think we should be able to parametrize with ``frame_or_series``?

Reply:
Yeah, but it appears the file structure was intended to separate `test/frame/method` from `test/series/methods`? Also `axis=1` shouldn't be applicable for `Series`

Reply:
thanks @mroeschke @phofl for the suggestions.
Have added the testcases for Series in a new file under `test/series/methods`
Let me know if any changes / alternatives to try. TIA